e2e/cmdenvvars/cacheDir.go

```diff
@@ -0,0 +1,83 @@
+// This software is licensed under a 3-clause BSD license. Please consult the
```

Comment:
Missing the copyright year thing

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
// This software is licensed under a 3-clause BSD license. Please consult the
```